Member mismatch in moveit_task_constructor_visualization build failure

### Description
Build failing with `moveit2_tutorials` and its dependencies  on humble/focal while trying to build from source. Seems like a version mismatch but I can't figure out which package/commit this could be related to. An older issue mentioned `srdfdom` to be on the ${ROS_DISTRO} branch but I have that already on `release/humble/srdfdom/2.0.4-1`. Any suggestions where to look?

### Environment
* ROS Distro: Humble
* OS Version: Ubuntu 20.04
* Source build: `moveit_core` and `moveit2_tutorials` on `humble`, other deps generated from `rosinstall_generator` or on `ros2` branch

### Console output

```
[Processing: moveit_setup_app_plugins, moveit_setup_controllers, moveit_setup_core_plugins, moveit_setup_srdf_plugins, moveit_task_constructor_visualization]    
--- stderr: moveit_task_constructor_visualization                                                                                                                
/home/user/moveit2/src/moveit_task_constructor/visualization/motion_planning_tasks/src/task_panel.cpp: In constructor ‘moveit_rviz_plugin::TaskPanel::TaskPanel(QWidget*)’:
/home/user/moveit2/src/moveit_task_constructor/visualization/motion_planning_tasks/src/task_panel.cpp:94:89: error: ‘idClicked’ is not a member of ‘QButtonGroup’
   94 |  connect(d->tool_buttons_group, static_cast<void (QButtonGroup::*)(int)>(&QButtonGroup::idClicked), d->stackedWidget,
      |                                                                                         ^~~~~~~~~
make[2]: *** [motion_planning_tasks/src/CMakeFiles/motion_planning_tasks_rviz_plugin.dir/build.make:245: motion_planning_tasks/src/CMakeFiles/motion_planning_tasks_rviz_plugin.dir/task_panel.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:450: motion_planning_tasks/src/CMakeFiles/motion_planning_tasks_rviz_plugin.dir/all] Error 2
make: *** [Makefile:146: all] Error 2
---
Failed   <<< moveit_task_constructor_visualization [3min 30s, exited with code 2]
Aborted  <<< moveit_setup_app_plugins [2min 49s]                                                                                                                
Aborted  <<< moveit_setup_core_plugins [3min 19s]                                                                                                                
Aborted  <<< moveit_setup_controllers [5min 27s]                                                                                                             
Aborted  <<< moveit_setup_srdf_plugins [6min 25s]                                              

Summary: 117 packages finished [57min 33s]
  1 package failed: moveit_task_constructor_visualization
  4 packages aborted: moveit_setup_app_plugins moveit_setup_controllers moveit_setup_core_plugins moveit_setup_srdf_plugins
  42 packages had stderr output: angles controller_manager eigen_stl_containers generate_parameter_library generate_parameter_library_py geometric_shapes ignition_cmake2_vendor ignition_math6_vendor image_common joint_state_publisher joint_state_publisher_gui launch_param_builder libcurl_vendor moveit_common moveit_configs_utils moveit_planners moveit_plugins moveit_resources moveit_resources_fanuc_description moveit_resources_fanuc_moveit_config moveit_resources_panda_description moveit_resources_panda_moveit_config moveit_resources_pr2_description moveit_resources_prbt_moveit_config moveit_resources_prbt_pg70_support moveit_resources_prbt_support moveit_ros moveit_runtime moveit_setup_srdf_plugins moveit_task_constructor_visualization octomap python_qt_binding ros2_control ros2_control_test_assets ros2_controllers ros2controlcli rviz_assimp_vendor rviz_ogre_vendor urdfdom_py vision_opencv xacro yaml_cpp_vendor
  3 packages not processed
```

--- a/visualization/motion_planning_tasks/src/task_panel.cpp
+++ b/visualization/motion_planning_tasks/src/task_panel.cpp
@@ -91,8 +91,13 @@ TaskPanel::TaskPanel(QWidget* parent) : rviz_common::Panel(parent), d_ptr(new Ta
 	Q_D(TaskPanel);
 
 	// sync checked tool button with displayed widget
+#if QT_VERSION >= QT_VERSION_CHECK(5, 15, 0)
 	connect(d->tool_buttons_group, static_cast<void (QButtonGroup::*)(int)>(&QButtonGroup::idClicked), d->stackedWidget,
 	        [d](int index) { d->stackedWidget->setCurrentIndex(index); });
+#else
+	connect(d->tool_buttons_group, static_cast<void (QButtonGroup::*)(int)>(&QButtonGroup::buttonClicked),
+	        d->stackedWidget, [d](int index) { d->stackedWidget->setCurrentIndex(index); });
+#endif
 	connect(d->stackedWidget, &QStackedWidget::currentChanged, d->tool_buttons_group,
 	        [d](int index) { d->tool_buttons_group->button(index)->setChecked(true); });
 

